web-components/src/components/menu-overlay/MenuOverlay.ts

```diff
@@ -73,6 +73,7 @@ export namespace MenuOverlay {
     @property({ type: String }) ariaRole: Role = "menu";
     @property({ type: String }) ariaLabel = "";
     @property({ type: Boolean, attribute: "is-date-picker" }) isDatePicker = false;
+    @property({ type: Boolean, attribute: "noToggleOnClearInput" }) noToggleOnClearInput = false; // This will prevent the overlay from toggling when the input field is cleared
```

Comment:
Very minor. noToggleOnClearInput being set to false implies there is toggle on clearing. Maybe rename to just
toggleOnClearInput = false;

Reply:
Sure, will do!